sky/packages/sky/lib/rendering/object.dart

```diff
@@ -533,20 +660,39 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
     markNeedsPaint();
     assert(parent == this.parent); // TODO(ianh): Remove this once the analyzer is cleverer
   }
-  bool get sizedByParent => false; // return true if the constraints are the only input to the sizing algorithm (in particular, child nodes have no impact)
-  void performResize(); // set the local dimensions, using only the constraints (only called if sizedByParent is true)
+
+  /// Whether the constraints are the only input to the sizing algorithm (in particular, child nodes have no impact)
+  ///
+  /// Returning false is always correct, but returning true can be more
+  /// efficient in some siutations.
```

Comment:
Correct the spelling of situations and elaborate on what some of those might be...